src/ontology/uberon-edit.obo

```diff
@@ -226353,6 +226354,29 @@ is_a: UBERON:7500094 ! tubercle of calcaneus
 property_value: http://purl.org/dc/elements/1.1/date 2021-08-04T00:27:56Z xsd:dateTime
 property_value: term:editor http://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500098
+name: sagittal crest of skull
+def: "A ridge of bone running lengthwise along the midline of the top of the skull of many mammalian and reptilian skulls and serves as an attachment site for the temporalis muscle[WP, modified]." [Wikipedia:Sagittal_crest]
+synonym: "sagittal ridge" RELATED []
+xref: Wikipedia:Sagittal_crest
+is_a: UBERON:0005913 ! zone of bone organ
+relationship: part_of UBERON:0003113 ! dermatocranium
+property_value: depicted_by Paranthropus:aethiopicus.JPG xsd:string
+property_value: external_ontology_notes "Compare with AAO:0000384 parasagittal crest (UBERON:0013402)" xsd:string {external_ontology="AAO"}
+property_value: function_notes "The sagittal crest serves primarily for attachment of the temporalis muscle, which is one of the main chewing muscles" xsd:string
+property_value: http://purl.org/dc/elements/1.1/date 2021-11-09T21:41:47Z xsd:dateTime
+property_value: taxon_notes "Skulls of some dinosaur species, including tyrannosaurs, possessed well developed sagittal crests. Among mammals, dogs, cats, lions, and many other carnivores have sagittal crests, as do some leaf eaters, incluing tapirs and some apes. Sagittal crests are found in robust great apes, and some early hominins. Prominent sagittal crests are found among male gorillas and orangutans, and do occur but only rarely in male chimpanzees such as Bili Apes" xsd:string
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500099
+name: sagittal crest
+def: "A ridge or similar projection rising above the surface of an anatomical structure along the sagittal (anterior-posterior) axis." [https://orcid.org/0000-0003-2699-3066]
+is_a: UBERON:4200133 ! crest
+property_value: http://purl.org/dc/elements/1.1/date 2021-11-09T21:49:16Z xsd:dateTime
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
```

Comment:
something is not configured right, `term:editor` is not right

Reply:
should it just be "database_cross_reference"?